src/main/java/com/training/web/pages/rccl/ProfilePage.java

```diff
@@ -0,0 +1,12 @@
+package com.training.web.pages.rccl;
+
+import com.codeborne.selenide.SelenideElement;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class ProfilePage {
+    public SelenideElement uiFullName= $x("//h2[contains(text(),'Audrey Poole')]");
+
+    public String getUiFullName(){
```

Comment:
why Ui in the method name. It is understood that POM will return element from Ui